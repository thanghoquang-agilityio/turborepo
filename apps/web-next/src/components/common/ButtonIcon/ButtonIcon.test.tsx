import { render } from '@testing-library/react'

import { ButtonIcon } from '.'

describe('ButtonIcon', () => {
  it('Renders empty title and subtitle match snapshot', () => {
    const { container } = render(<ButtonIcon disabled />)
    expect(container).toMatchSnapshot()
  })
})
