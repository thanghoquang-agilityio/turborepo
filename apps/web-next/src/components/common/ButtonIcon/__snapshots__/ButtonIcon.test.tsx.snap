// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ButtonIcon Renders empty title and subtitle match snapshot 1`] = `
<div>
  <button
    aria-label="Button Icon"
    class="z-0 group relative inline-flex items-center justify-center box-border appearance-none select-none whitespace-nowrap subpixel-antialiased overflow-hidden tap-highlight-transparent data-[pressed=true]:scale-[0.97] outline-none data-[focus-visible=true]:z-10 data-[focus-visible=true]:outline-2 data-[focus-visible=true]:outline-focus data-[focus-visible=true]:outline-offset-2 h-10 text-small [&>svg]:max-w-[theme(spacing.8)] transition-transform-colors-opacity motion-reduce:transition-none data-[hover=true]:opacity-hover gap-2.5 font-semibold bg-background-300 text-primary-300 rounded-[100%] lg:min-w-[50px] lg:min-h-[50px] lg:p-[15px] md:max-h-[40px] md:max-w-[40px] md:p-[10px] max-w-[30px] max-h-[30px] p-[7px] min-w-[30px] cursor-default"
    disabled=""
    type="button"
  />
</div>
`;
